emmaa_service/api.py

```diff
@@ -824,7 +825,10 @@ def _make_badges(evid_count, json_link, path_count, cur_counts=None):
          'symbol': None, 'title': 'Evidence count for this statement',
          'loc': 'right'},
         {'label': 'paths', 'num': path_count, 'symbol': '\u2691',
-         'color': '#0099ff', 'title': 'Number of paths with this statement'}]
+         'color': '#0099ff', 'title': 'Number of paths with this statement'},
+        {'label': 'belief', 'num': belief, 'color': '#ffc266',
+         'symbol': None, 'title': 'Belief score for this statement',
+         'loc': 'right'}]
```

Comment:
These badges could use a pixel or two of padding between them

Reply:
Good point. The badges are actually handled by Vue components outside EMMAA so that would have to be changed there (the same applies to the tooltips suggestion). EMMAA only handles the "content" of the badges.

Reply:
Okay, then not appropriate for this PR. Where could I add an issue for this? In https://github.com/indralab/indralab-vue?

Reply:
Yes, that is where that code lives